jberet-se-bom/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ Copyright (c) 2021 Red Hat, Inc. and/or its affiliates.
+
+ This program and the accompanying materials are made
+ available under the terms of the Eclipse Public License 2.0
+ which is available at https://www.eclipse.org/legal/epl-2.0/
+
+ SPDX-License-Identifier: EPL-2.0
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jberet-parent</artifactId>
+        <groupId>org.jberet</groupId>
+        <version>1.4.5.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.jberet</groupId>
+    <artifactId>jberet-se-bom</artifactId>
+    <packaging>pom</packaging>
+    <name>Jberet-BOM</name>
+    <description>Jberet Java Standard Edition BOM</description>
+    <properties>
+        <!-- Jberet SE 1.4 Overriding BOM Dependency versions  -->
+        <version.org.jberet>1.4.4.Final</version.org.jberet>
+        <version.org.jberet.support>1.3.4.Final</version.org.jberet.support>
+        <version.org.spec.javax.batch>2.0.0.Final</version.org.spec.javax.batch>
+        <version.org.spec.javax.transaction>2.0.0.CR1</version.org.spec.javax.transaction>
```

Comment:
org.spec.javax... seems odd. Can you use the same property name as in jberet-parent, such as

version.org.jboss.spec.javax.batch.jboss-batch-api_1.0_spec

version.org.jboss.spec.javax.transaction.jboss-transaction-api_1.2_spec